docs/installation/bootstrap.rst

```diff
@@ -49,6 +49,8 @@ Configuration
           cidr: <CIDR-notation>
           ingress:
             ip: <IP-for-ingress>
+          metalLB:
+            enabled: true
```

Comment:
```suggestion
            enabled: <boolean>
```

Reply:
No no no you must enable it !! It's mandatory (joke)
Sure I will change it